packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -201,6 +209,23 @@ class Tooltip extends StatefulWidget {
   /// pointer exits the widget.
   final Duration? showDuration;
 
+  /// The [TooltipTriggerMode] that will show the tooltip.
+  ///
+  /// If this property is null, then [TooltipThemeData.triggerMode] is used.
+  /// If [TooltipThemeData.triggerMode] is also null, the default mode is
+  /// [_kTooltipTriggerMode]
```

Comment:
You do not have to reference this variable name. You can just say what the value is by default.

Reply:
Do I have to though? I prefer to keep the reference, if accepted.

Reply:
It's a private variable. It will not make sense to the user reading the documentation, it should just say what the default value is.

Reply:
Commented about it being private below; I want to fix that.